Seagate/cortx-utils
# Cortx-29731:  Event Schema shows Event id and Timestamp in floating values

# Problem Statement
- https://jts.seagate.com/browse/CORTX-29731

# Design
- Timestamp float is may other component required, only add decimal value in event_id
- 
# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]:  Y

# Testing
- Are test cases updated along with code changes due to Enhancements/Bugs [Y/N]: Y
- Confirm that new test cases are added to regression and sanity plan files and relevant feature plan files [Y/N]:
- Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]:  Y
- Confirm Testing was performed with installed RPM/K8s deployment [Y/N]:  Y

# Review Checklist 
####  Before posting the PR please ensure:

Test Results : 
sh-4.4# vi "/usr/lib/python3.6/site-packages/cortx/utils/event_framework/event.py"
sh-4.4# python3 test_health_event.py
.1650942515a327b56899724d6d876653eabf5d8952
..
----------------------------------------------------------------------
Ran 3 tests in 0.001s

OK

Sample/local test output
 python3 health.py
Before setting Payload : 
{"header": {"version": "1.0", "timestamp": "1650943256.3843129", "event_id": "165094325686708d4871964c2997b5877f437a5c84"}, "payload": {"source": "", "cluster_id": "", "site_id": "", "rack_id": "", "storageset_id": "", "node_id": "", "resource_type": "", "resource_status": "", "specific_info": ""}}

Setting payload : 
{"header": {"version": "1.0", "timestamp": "1650943256.3876772", "event_id": "1650943256f045f9f6c7c247978106d4b80b2c4a6d"}, "payload": {"source": "s1", "cluster_id": "1234", "site_id": "1", "rack_id": "1", "storageset_id": "1", "node_id": "3", "resource_type": "node", "resource_status": "ok", "specific_info": {"a": "a1", "b": true}}}
